include/convex_bodies/orderpolytope.h

```diff
@@ -216,21 +208,16 @@ class OrderPolytope {
     //Use LpSolve library
     std::pair<Point, NT> ComputeInnerBall()
     {
-       normalize();
-
-        // change entries of A, doing here as won't be required in
-        // optimized volume calculation of order-polytope
-        MT A = _A.rowwise().normalized();
+        normalize();
         std::pair<Point, NT> _innerball;
 
-        #ifndef VOLESTIPY   // as _A is never normalized in closed form
-            _innerball = ComputeChebychevBall<NT, Point>(A, b); // use lpsolve library
+        #ifndef VOLESTIPY
```

Comment:
VOLESTIPY is always undefined. Plz use only `ComputeChebychevBall<NT, Point>(A, b)`

Reply:
@TolisChal  is VOLESTIPY needed in the library now?

Reply:
https://github.com/GeomScale/volume_approximation/issues/148

Reply:
now the function returns just `ComputeChebychevBall<NT, Point>(A, b)`